### ISSUE TITLE
Largest files fail to generate preview

### ISSUE THREAD

**Issue by cjhendrix** [OCHA-DAP/hdx-ckan#3237]
Error is 

"{\"error_class\": \"PushingToPostgisException\", \"layer_id\": \"pre_846da015_d427_4279_aa63_37d4d50e1b65\", \"error_type\": \"None\", \"state\": \"failure\", \"message\": \"Problem while trying to push data to postgis\", \"type\": \"pushing-to-postgis-problem\"}",

**Comment by cjhendrix** [comment]
This may not necessarily be a blocker, but we would need to change the language of the FAQ and blog post after testing to see what the failure limit is on prod.

**Comment by cjhendrix** [comment]
@alexandru-m-g So did we just close this one on the call with Serban?

**Comment by teodorescuserban** [comment]
Yep closed

Joi, 30 iul. 2015, 15:48, cjhendrix notifications@github.com a scris:

**Comment by cjhendrix** [comment]
On test, still getting the error on a dropbox link:  http://test-data.hdx.rwlabs.org/api/3/action/package_show?id=geotest01

Unable to complete upload of the same file (chrome gets to 1 or 2 percent, then resets).

**Comment by alexandru-m-g** [comment]
1.  I was able to complete a big upload on testing. Check https://test-data.hdx.rwlabs.org/dataset/geotest01
2.  The geoprocessing doesn't really start ( on test). Since 'test' is on blackmesh now I don't have access to debug it myself. I'll need help from @teodorescuserban .
3.  I've committed a small fix: that's related to the "shape_info" attribute not being populated in some cases for new resources before the processing starts. Unfortunately, it has nothing to do with the success or failure of the transforrmation ( nor with point 2).
